ddocs/users-meta-db/users-meta/views/device_by_user/map.js

```diff
@@ -27,7 +27,34 @@ function(doc) {
           cht: doc.metadata.versions && doc.metadata.versions.app,
           settings: doc.metadata.versions && doc.metadata.versions.settings,
         },
+        storage: {
+          free: doc.device &&
+            doc.device.deviceInfo &&
+            doc.device.deviceInfo.storage &&
+            doc.device.deviceInfo.storage.free,
+          total: doc.device &&
+            doc.device.deviceInfo &&
+            doc.device.deviceInfo.storage &&
+var deviceInfo = doc.device && doc.device.deviceInfo;
+ emit([doc.metadata.user, doc.metadata.deviceId], {
```

Comment:
Here you are emitting twice. I think you just added the code snippet I linked instead of incorporating it in the existent code.